src/adapters/source-adapters/CoinbaseOracleSourceAdapter.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.17;
+
+import {DecimalLib} from "../lib/DecimalLib.sol";
+import {IAggregatorV3Source} from "../../interfaces/chainlink/IAggregatorV3Source.sol";
+import {DiamondRootOval} from "../../DiamondRootOval.sol";
+
+/**
+ * @title CoinbaseOracleSourceAdapter
+ * @notice A contract to read data from CoinbaseOracle and standardize it for Oval.
+ * @dev Can fetch information from CoinbaseOracle source at a desired timestamp for historic lookups.
+ */
+abstract contract CoinbaseOracleSourceAdapter is DiamondRootOval {
```

Comment:
should we just call this file `CoinbaseSourceAdapter`? we don't call others things like `PythOracleSourceAdapter` or `ChainlinkOracleSourceAdapter`; they are `PythSourceAdapter` and `ChainlinkSourceAdapter`

Reply:
+1